options.go

```diff
@@ -410,6 +410,15 @@ type BufferOption interface {
 	setBufferOpt(bo *bufferOpts)
 }
 
+// IntersectsOption is an option passed to Geometry.Intersects()
+//
+//
+// Available options are:
+//  - ErrLogger
+type IntersectsOption interface {
+	setIntersectsOpt(bo *bufferOpts)
```

Comment:
you should add and use a new `intersectsOpts` structure, instead of re-using `bufferOpts`

Reply:
Copy paste error, done for next commit.